README.md

```diff
@@ -35,6 +35,61 @@ https://www.wowza.com/testplayers?src=https://wse-demo.wowza.com/whisper/myStrea
 ## Configuration
 See the included Application.xml files for the module configurations.
```

Comment:
It may be helpful to link to the Application.xml files in the repo here, so there's no confusion where to find these files.